Pages/Setting-Section-Page-Numbers-With-Custom-Styles/.NET/Setting-Section-Page-Numbers-With-Custom-Styles/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Syncfusion.Pdf;
+using Syncfusion.Pdf.Graphics;
+
+//Create a new document.
+PdfDocument document = new PdfDocument();
+//Add the section.
+PdfSection section = document.Sections.Add();
+//Set the font.
+PdfFont font = new PdfStandardFont(PdfFontFamily.Helvetica, 12);
+//Create section page number field.
+PdfSectionPageNumberField sectionPageNumber = new PdfSectionPageNumberField();
+//Set the page number style as LowerRoman
+sectionPageNumber.NumberStyle = PdfNumberStyle.LowerRoman;
+sectionPageNumber.Font = font;
+//Draw the sectionPageNumber in section.
+for (int i = 0; i < 3; i++)
+{
+    PdfPage page = section.Pages.Add();
+    sectionPageNumber.Draw(page.Graphics);
```

Comment:
@sameerkhan001 , The page number field must be drawn on the footer content. So change the code and let me know

Reply:
Updated code